kratos/utilities/parallel_utilities.h

```diff
@@ -11,29 +11,27 @@
 //                   Denis Demidov
 //
 
-#if !defined(KRATOS_PARALLEL_UTILITIES_H_INCLUDED )
-#define  KRATOS_PARALLEL_UTILITIES_H_INCLUDED
-
+#if !defined(KRATOS_PARALLEL_UTILITIES_H_INCLUDED)
+#define KRATOS_PARALLEL_UTILITIES_H_INCLUDED
 
 // System includes
-#include<iostream>
-#include<array>
-#include<vector>
-#include<tuple>
-#include<cmath>
-#include<limits>
-#include<omp.h>
-
+#include <iostream>
+#include <array>
+#include <vector>
+#include <tuple>
+#include <cmath>
+#include <limits>
 #include <future>
 #include <thread>
-#include "utilities/reduction_utilities.h"
 
 // External includes
-
+#include <omp.h>
```

Comment:
could we include the openmp_utils.h instead?

Reply:
But thats the one we want to get rid of why including it here?

I'd rather do it the other way round in case we need it for backward compatibility

Reply:
lets discuss this separately so that I can go ahead :)